.github/workflows/lint.yml

```
@@ -18,6 +18,6 @@ jobs:
           go-version-file: go.mod
 
       - name: Run linter
-        uses: golangci/golangci-lint-action@v6
+        uses: golangci/golangci-lint-action@v7
         with:
           version: v1.63.4
```

```suggestion
          version: v2.1.6
```

I think we need to bump the golangci-lint version here as well.